back/engines/free/email_campaigns/spec/acceptance/campaigns_spec.rb

```diff
@@ -32,22 +32,10 @@
       json_response = json_parse(response_body)
       expect(json_response[:data].size).to eq 5
     end
-
-    example 'List all manual campaigns' do
-      do_request(campaign_names: ['manual'])
-      json_response = json_parse(response_body)
-      expect(json_response[:data].size).to eq 3
-    end
-
-    example 'List all non-manual campaigns' do
-      do_request(without_campaign_names: ['manual'])
-      json_response = json_parse(response_body)
-      expect(json_response[:data].size).to eq 2
-    end
```

Comment:
Is there no reason for this endpoint to work?

I thought the `Automated emails` tab was supposed to stay, does it not use this endpoint?
![image](https://user-images.githubusercontent.com/329308/213676735-48842ae6-24d6-480d-ab61-1e37280fd359.png)

Reply:
It should still work. I just removed the test because it felt redundant with "List all campaigns" since all campaigns are "non-manual". I could actually put it back since it tests a query parameter that's not used in the other one.

Reply:
![image](https://user-images.githubusercontent.com/209268/213724218-efbe78e3-6d3c-43ab-bfb5-4ed3e326dcae.png)